NinchatSDKSwift/Implementations/Managers/Session Manager/NINChatSessionManagerImpl.swift

```diff
@@ -199,6 +199,12 @@ extension NINChatSessionManagerImpl {
             guard let `self` = self else { return }
             
             if event == .sessionCreated {
+                if let err = params.error as? NinchatError {
+                    if err.type == "user_not_found" {
+                        self.endSession()
+                    }
+                    completion(credentials, nil, error); return
+                }
```

Comment:
Can you please help me understand the code a bit ?
I am under the impression that this code block will go on` if event == .sessionCreated ` level instead of under it. I meant,

Reply:
I think this should go inside the `if event == .error` block.

Reply:
> 

Thanks, yes I wanted to mean that too

Reply:
Thanks for comments, I had realized Android code wrongly